src/codeql.ts

```diff
@@ -81,13 +81,17 @@ libraryPathDependencies: codeql-${language}`
 }
 
 async function downloadDatabase(
-  token: string,
+  signedAuthToken: string,
   repoId: number,
-  language: string
+  language: string,
+  pat?: string
```

Comment:
I find it odd that `signedAuthToken ` and `pat` are not equal citizens of this argument list. Personally I'd put them next to each other and make them both nullable / undefinable.

I realise to us, using a pat is the weird case that only happens in the integration test, but in the code it's just as valid a case as a signed auth token. That's my reasoning anyway.

Reply:
Sure, I had a similar feeling, I'll update it as you suggest.